Makefile

```diff
@@ -0,0 +1,71 @@
+all: run
+
+# Set these to your own values first
+JUPYTER_TOKEN=secr3t
+DOCKERHUB_ID=ibmosquito

```

Comment:
@tahira-tariq just for the sake of consistence could you change this to `DOCKER_HUB_ID=ibmosquito`? I might have let some of these slip by accidentally but we more broadly use `DOCKER_HUB_ID` in the other services

Reply:
@tahira-tariq I actually just realized the way that's written `DOCKER_HUB_ID` can't be overwritten at all ... can you change it to `DOCKER_HUB_ID ?= ibmosquito` similar to how it is in `web-helloworld-python`

https://github.com/open-horizon-services/web-helloworld-python/blob/main/Makefile#L3